### DISCUSSION TITLE
这个仓库主要用来做什么?

### DISCUSSION THREAD

**Discussion by libeng** [adk23333/tieba-admin#10] [Q&A] [answered]
将某个吧主或者小吧的账号权限再分发给不同的用户?

**Comment by adk23333** [comment, marked as answer]
这是其中之一的功能，我将实现两个插件来搞定[reviewer](https://github.com/Starry-OvO/aiotieba-reviewer)的主要功能，后续看反馈添加插件，或者大家提交，又或者自行开发插件。目前还在持续完善阶段，无法正常使用。

主要还是提供接口，感兴趣的可以帮忙开发管理app

**Comment by n0099** [comment]
https://github.com/search?q=owner%3AStarry-OvO+involves%3Aadk23333&type=issues
